frog/imports/client/Wiki/index.js

```diff
@@ -347,7 +358,7 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
 
   changeTitle = (pageId, newPageTitle) => {
     changeWikiPageTitle(this.wikiDoc, pageId, newPageTitle);
-    const instanceId = this.props.match.params.instance;
+    const instanceId = this.props.match?.params.instance;
```

Comment:
What happens if instanceId is undefined?

Reply:
It would result in an error, it was an inadvertent error. This is a good reason to not use maybe types to control the logic.

Reply:
Made the respective changes